charts/pulsar/templates/log4j-configmap.yaml

```diff
@@ -0,0 +1,143 @@
+#
```

Comment:
It seems that this log4j-configmap file is used by multiple components. If we change the log setting in this log4j configmap, it will affect all the components. This doesn't make sense to me. We should be able to configure log4j for individual components.

Reply:
Thanks @sijie will fix it